_redirects/guides/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -0,0 +1,5 @@
+[source,markdown]
+---
+permalink: /guides/security-authorization-of-web-endpoints-reference.adoc/index.html
```

Comment:
The `.adoc' file extension bit is not needed.
```suggestion
permalink: /guides/security-authorization-of-web-endpoints-reference/index.html
```
[See demo  slides & example PR]